pkg/apis/operator/v1alpha1/tektonaddon_defaults.go

```diff
@@ -56,6 +56,6 @@ func setAddonDefaults(addon *Addon) {
 
 	// by default enable pac
 	if addon.EnablePAC == nil {
```

Comment:
Is it for backward compatibility ?

Reply:
yes will be, I need to handle a few checks in tektonconfig.

Reply:
will be doing after this.